panels/config/ha-panel-config.html

```diff
@@ -49,6 +50,14 @@
           hass='[[hass]]'
         ></ha-config-section-core>
 
+        <template is='dom-if' if='[[computeIsThemesLoaded(hass)]]'>
```

Comment:
hass.themes.themes is an Object not an array. So no length, and empty object is a boolean True.